rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2021-11-01
```

Comment:
Once you've renamed the other toolchain file, this one can be safely removed.

Reply:
Done :+1: